extensions/ShowierData9978/html.js

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable require-await */ 
```

Comment:
Metadata must be at the very start of the extension's script, this needs to be moved